modules/pytket-honeywell/tests/backend_test.py

```diff
@@ -235,6 +235,26 @@ def test_classical() -> None:
     assert b.get_counts(c, 10)
 
 
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
+def test_postprocess() -> None:
+    b = HoneywellBackend("HQS-LT-S1-APIVAL")
+    assert b.supports_contextual_optimisation
+    c = Circuit(2, 2)
+    c.add_gate(OpType.PhasedX, [1, 1], [0])
+    c.add_gate(OpType.PhasedX, [1, 1], [1])
+    c.add_gate(OpType.ZZMax, [0, 1])
```

Comment:
why not do the same bell test and check the outsput shots?

Reply:
I don't think this would work, as `APIVAL` doesn't actually do a simulation?